### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_CodeEditingRedefined_Keyboard: Tooltip is missing for non text content other platform and insiders edition button.

### ISSUE THREAD

**Issue by mspatil110** [microsoft/vscode-docs#3588]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#CELAP1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.1.1;#Win10;#Visual Studio Code;#CELAP1_code;#ColorContrast;

Environment Details:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/python/python-tutorial
Windows Version: Windows10
Browser version: Microsoft Edge Version 81.0.416.60 (Official build) beta (64-bit)

Repro Steps:
1)Launch the application url :https://code.visualstudio.com/
2)Visual Studio Code will get open.
3)Go through "Code editing Redefined".
4)Navigate up to other platform and insiders edition button and observed that tool tip is present for this non text content or not.

Actual:
Tooltip is missing for non text content other platform and insiders edition button.

Expected:
Tooltip "other platform and insiders edition" should be present for non text content other platform and insiders edition button.

User Impact:
Keyboard only user will face difficulty to understand the button without tooltip.

Attachment link for Reference:
[11042_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_CodeEditingRedefined_Keyboard.pptx](https://github.com/microsoft/vscode-docs/files/4510764/11042_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_CodeEditingRedefined_Keyboard.pptx)

**Comment by gregvanl** [comment]
Tooltip added

![image](https://user-images.githubusercontent.com/12818376/82613159-7b49be80-9b79-11ea-939d-be8a777b65f7.png)

**Comment by khansahab15** [comment]
GitHubTags:#P1_A11y_April2020

**Comment by ManishaC1** [comment]
Verified in below environment.
URL: https://code.visualstudio.com/
Issue has been fixed.
Github tag:#Closed

https://user-images.githubusercontent.com/105198219/216907327-27269385-2a98-4d2e-942b-932ac17ac4d4.mp4

**Comment by kalpanah1** [comment]
#closed;
Verified the issue in the below environment. The issue is fixed. Adding the attachment
![Bug_3588_Fixed](https://user-images.githubusercontent.com/94966876/217799886-1e24b493-c59d-47b2-a9c7-dc7caefa6aeb.png)

**Comment by fsteffi** [comment]
githubtags:#P1_A11y_April2020;

**Comment by mstechie** [comment]
GitHubTags:#P1_A11y_code.visualstudio.com_Apr2020
